assets/src/dashboard/templates/getTemplates.js

```diff
@@ -61,11 +62,19 @@ export default async function loadTemplates(imageBaseUrl) {
     'wellbeing',
   ];
 
-  const result = await Promise.all(
-    templates.map(async (title) => {
+  const trackTiming = getTimeTracker('load', 'dashboard', 'Templates');
+
+  const templates = await Promise.all(
+    templateNames.map(async (title) => {
       return [title, await loadTemplate(title, imageBaseUrl)];
     })
   );
 
-  return Object.fromEntries(result);
+  const result = Object.fromEntries(templates);
+
+  await trackTiming();
```

Comment:
Are we sure we want to be awaiting the tracking ping (here and below)? IMO we shouldn't block any user flow on it.

Reply:
Good point, removed now.